deploifai/data/list.py

```diff
@@ -0,0 +1,28 @@
+import click
+from deploifai.context import (
+    pass_deploifai_context_obj,
+    DeploifaiContextObj,
+    is_authenticated,
+    project_found,
+)
+from deploifai.api import DeploifaiAPIError
+
+
+@click.command("list")
+@pass_deploifai_context_obj
+@is_authenticated
+def list_data(context: DeploifaiContextObj):
```

Comment:
As discussed, listing priority should be
1. command option for project name
2. project id in local config (if found)
3. current workspace in global config (if project not found in local config)